activate.sh

```diff
@@ -46,3 +46,7 @@ if ! [ -f /root/.ssh/id_rsa ]; then
 fi
 [ -d /var/lib/misc/ssh-public-key  ] || mkdir -p /var/lib/misc/ssh-public-key
 cp /root/.ssh/id_rsa.pub /var/lib/misc/ssh-public-key
+
+# Create MariaDB system user, bsc#1034885
+groupadd -g 331 mysql
+useradd -r -u 331 -g mysql -c "MySQL database admin" -d /var/lib/mysql -s /bin/false mysql
```

Comment:
Group does not have a gid in the package so you cannot know in advance which will be choosen in the mariadb image, but you can be certain about the uid 60 .